godtools/App/Features/ToolsFilter/Presentation/ToolFilterLanguageSelectionView/ToolFilterLanguageSelectionView.swift

```diff
@@ -13,10 +13,12 @@ struct ToolFilterLanguageSelectionView: View {
     private static let ultraLightGrey = Color.getColorWithRGB(red: 246, green: 246, blue: 246, opacity: 1)
     
     @ObservedObject private var viewModel: ToolFilterLanguageSelectionViewModel
-    
-    init(viewModel: ToolFilterLanguageSelectionViewModel) {
+    @ObservedObject private var selectedLanguage: CombineObservableValue<LanguageFilterDomainModel>
```

Comment:
Does `CombineObservableValue` only trigger changes if it's hooked up to a view?  Or would it be possible for it to live in the viewModel?  If it has to live in the view, I'm wondering what the difference is between this and using `@Published`/`@Binding` in tandem.

Reply:
For example, `DashboardViewModel` has a property `@Published var currentTab: Int` that gets passed into `DashboardTabBarItemView` as a `@Binding`, and the tab value can get updated from either end.  Seems kind of like the same idea?

Reply:
Hey @rachaelblue now that you mention it I think ViewModel would be more appropriate for MVVM.  Is that what you were thinking?

Reply:
Hmm I guess Binding would also work on the View with Published?  In that case CombineObservableValue wouldn't be necessary on the View.  Is that what you were thinking here?

Reply:
That's right Binding would be a better option and redundant to use CombineObservableValue.  Let me try Binding again, I remember I did get an error, just can't exactly remember what it stated.  I believe it had to do with scope since I was passing it through the Flow.

Reply:
Yeah, I was thinking as far as MVVM, the selectedLanguage should probably live in the viewModel.  I'm pretty sure that was my big issue in the beginning, that I didn't want to pass the value separately into the view.  Although I don't remember dealing with it needing to pass through the flow 🤔

Reply:
In my test I was passing CombineObservableValue from ToolsViewModel through the Flow to the Selection Views.

Let me try another approach using Binding on the Selection views and a Published on the ViewModel.  I'm curious what that might look like.